library/firewall_lib.py

```diff
@@ -541,6 +541,23 @@ def parse_forward_port(module, item):
     return (_port, _protocol, _to_port, _to_addr)
 
 
+def parse_firewalld_conf(am, firewalld_conf):
```

Comment:
```suggestion
def parse_firewalld_conf(am, firewalld_conf):
```
suggest a more descriptive name like `check_allow_zone_drifting` or something similar

Reply:
I can make a `check_allow_zone_drifting` function to be used within `parse_firewalld_conf`, but  I prefer to keep `parse_firewalld_conf` to check other arguments for `firewalld_conf` if they are added (ex. FirewallBackend).

Reply:
ok - it doesn't look like `parse_firewalld_conf` is actually parsing firewalld.conf - so I'm not sure what is the intention of `parse_firewalld_conf` or how to make it more generalized for other parameters

Reply:
Does the most recent commit solve the issue?

Reply:
> ok - it doesn't look like `parse_firewalld_conf` is actually parsing firewalld.conf - so I'm not sure what is the intention of `parse_firewalld_conf` or how to make it more generalized for other parameters

I see what you mean, this function checks the parameter `firewalld_conf`, should I change the name of the parameter to something like just `conf` so it is not conflated with the configuration file?

Then "parse" was probably a poor choice of words, so I changed the function name as well.